files/qtip2/update.json

```diff
@@ -1,8 +1,8 @@
 {
-  "packageManager": "github",
+  "packageManager": "npm",
   "name": "qtip2",
-  "repo": "qtip2/bower",
+  "repo": "qtip2/qtip2",
   "files": {
-    "include": ["*.js", "*.css", "*.map", "./basic/**/*"]
+    "include": ["dist/**/*.js", "dist/**/*.css", "dist/**/*.map"]
```

Comment:
If you want to include all files inside dist/ you can simply set `basePath` to `dist/` without any include parameters